packages/fondue/package.json

```diff
@@ -103,8 +103,8 @@
         "chalk": "^5.3.0",
         "concat": "^1.0.3",
         "cross-env": "^7.0.3",
-        "cypress": "^13.6.4",
-        "cypress-real-events": "^1.12.0",
```

Comment:
I am pinning these versions as we want to share the same cache. We are not exactly doing this yet in the actions, but I didn't want to increase the scope and make changes into the existing `old-component` workflow